Header header
float32 value
string time
int32 track_number
bool control
